components.yaml

```diff
@@ -30,11 +30,11 @@ components:
     update-policy: tagged
     metadata: v3.4.1
   nmstate:
-    url: https://github.com/nmstate/kubernetes-nmstate
-    commit: a0d3d5658390fac730bafcc85506a0a681da98ac
-    branch: master
-    update-policy: tagged
-    metadata: v0.31.0
+    url: "https://github.com/nmstate/kubernetes-nmstate"
```

Comment:
since the bumper script running on components.yaml is removing the `""`, let's keep this without the quotes. 
please also change for the other nmstate params below..

Reply:
Done, thanks